src/charts/tooltip.js

```diff
@@ -672,6 +673,20 @@ define(function(require){
             return this;
         };
 
+        /**
+         * Gets or Sets the number format for the value displayed on the tooltip
+         * @param  {string} [_x=".2f"] Desired number format
```

Comment:
Sorry just noticed it. the `[_x=".2f"]` will show that ".2f" is the default value while the default is `null`. currently. In `minitooltip` the default is indeed `.2f` but it's `null` in this case.